### PR TITLE
User presence is now shown

### PR DESCRIPTION
**Pull request by olbapmar**
Hi y'all!

The purpose of this pull-request is to show info about the activity of the users, as it happens in the official app. These are my first steps with javascript, which means that there will surely be room for improvement, and I'm open to suggestions! 

image

image

Cheers!

### CODE REVIEW

File: main/main.js [hunk 1]
```diff
@@ -71,11 +71,24 @@ let chatListTimeoutObj;
...
-
     if (chatListTimeoutObj) {
       clearTimeout(chatListTimeoutObj);
     }
+
+    instagram.getPresence().then((presenceInfo) => {
+      for(let key in presenceInfo.user_presence){
+        for(let chat in chats){
+          if(chats[chat].users.length === 1 && chats[chat].users[0].pk == key){
+            chats[chat].presence = presenceInfo.user_presence[key].is_active? 0 :
```
Comment (ifedapoolarewaju) [inline main/main.js:82]: maybe we can as well set the `user_presence[key]` object itself? I feel the value `0` or `timestamp` can easily get confusing now and then.

**Comment by ifedapoolarewaju** [comment]
Thank you for this PR! This will be a really nice addition. I have dropped some comments on possible changes to make.

**Comment by olbapmar** [comment]
Hi! Thanks for your comments. I've made some changes following them :)

**Comment by ifedapoolarewaju** [comment]
thank you!

**Pull request merged**